docs/introduction/usage.md

```diff
@@ -5,27 +5,27 @@ title: Usage
 
 # Usage
 
-See full usage documentation by typing `cadl --help`:
+See full usage documentation by typing `typespec --help`:
 
 ```bash
->cadl --help
-Cadl compiler v0.36.1
+>typespec --help
+TypeSpec compiler v0.36.1
 
-cadl <command>
+typespec <command>
 
 Commands:
-  cadl compile <path>       Compile Cadl source.
-  cadl code                 Manage VS Code Extension.
-  cadl vs                   Manage Visual Studio Extension.
-  cadl format <include...>  Format given list of Cadl files.
-  cadl init [templatesUrl]  Create a new Cadl project.
-  cadl install              Install cadl dependencies
-  cadl info                 Show information about current Cadl compiler.
+  typespec compile <path>       Compile TypeSpec source.
+  typespec code                 Manage VS Code Extension.
+  typespec vs                   Manage Visual Studio Extension.
+  typespec format <include...>  Format given list of TypeSpec files.
+  tsp init [templatesUrl]  Create a new TypeSpec project.
+  typespec install              Install typespec dependencies
+  typespec info                 Show information about current TypeSpec compiler.
```

Comment:
`tsp` (x6)

Reply:
usage.md still has one.  `typespec <command>` -> `tsp <command>`.

Manual fix

Reply:
in upcoming PR.